.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,3 +30,7 @@ A clear and concise description of what you expected to happen.
 **Additional context**
 
 Add any other context about the problem here.
+
+**Testing**
+
+If this issue is a regression or something that should block release, please consider including a test as a [hydraJob]( https://github.com/NixOS/nix/blob/master/flake.nix#L396) so that it can be part of the [automatic checks](https://hydra.nixos.org/jobset/nix/master).
```

Comment:
```suggestion
If this issue is a regression or something that should block release, please consider including a test either in the [testsuite](https://github.com/NixOS/nix/tree/master/tests) or as a [hydraJob]( https://github.com/NixOS/nix/blob/master/flake.nix#L396) so that it can be part of the [automatic checks](https://hydra.nixos.org/jobset/nix/master).
```

(Hydra jobs are cool, but for small inexpensive stuff, just extending the testsuite is probably easier and cheaper)